consensus/polybft/stake/stake_manager.go

```diff
@@ -1,62 +1,59 @@
-package polybft
+package stake
 
 import (
-	"bytes"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"math/big"
-	"sort"
-	"strings"
 
 	"github.com/0xPolygon/polygon-edge/bls"
 	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	polychain "github.com/0xPolygon/polygon-edge/consensus/polybft/blockchain"
 	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/state"
+	polytypes "github.com/0xPolygon/polygon-edge/consensus/polybft/types"
```

Comment:
looks prettier to me

```suggestion
	polyTypes "github.com/0xPolygon/polygon-edge/consensus/polybft/types"
```

Reply:
From Golang [guidelines](https://go.dev/blog/package-names) on package name:

`Good package names are short and clear. They are lower case, with no under_scores or mixedCaps`.